Sources/Misc/Obsoletions.swift

```diff
@@ -548,18 +548,18 @@ public extension Purchases {
      * - Warning: This assumes your IAP implementation uses StoreKit 1.
      * - Warning: If you're using observer mode with StoreKit 2, configure the SDK with `configure(withAPIKey:appUserID:observerMode:storeKitVersion:)` passing in `.storeKit2` as the `storeKitVersion` and ensure that you call ``Purchases/handleObserverModeTransaction(_:)`` after making a purchase.
      */
-    @available(iOS, obsoleted: 1, message: """
-    If you're using observer mode with StoreKit 2, ensure that you call Purchases.handleObserverModeTransaction(result:) after making a purchase.
-    """, renamed: "configure(withAPIKey:appUserID:observerMode:storeKitVersion:)")
-    @available(tvOS, obsoleted: 1, message: """
-    If you're using observer mode with StoreKit 2, ensure that you call Purchases.handleObserverModeTransaction(result:) after making a purchase.
-    """, renamed: "configure(withAPIKey:appUserID:storeKitVersion:)")
-    @available(watchOS, obsoleted: 1, message: """
-    If you're using observer mode with StoreKit 2, ensure that you call Purchases.handleObserverModeTransaction(result:) after making a purchase.
-    """, renamed: "configure(withAPIKey:appUserID:observerMode:storeKitVersion:)")
-    @available(macOS, obsoleted: 1, message: """
-    If you're using observer mode with StoreKit 2, ensure that you call Purchases.handleObserverModeTransaction(result:) after making a purchase.
-    """, renamed: "configure(withAPIKey:appUserID:observerMode:storeKitVersion:)")
+    @available(iOS, obsoleted: 1,
+               message: "Explicitly setting the StoreKit version is now required.",
+               renamed: "configure(withAPIKey:appUserID:observerMode:storeKitVersion:)")
+    @available(tvOS, obsoleted: 1,
+               message: "Explicitly setting the StoreKit version is now required.",
+               renamed: "configure(withAPIKey:appUserID:storeKitVersion:)")
+    @available(watchOS, obsoleted: 1,
+               message: "Explicitly setting the StoreKit version is now required.",
+               renamed: "configure(withAPIKey:appUserID:observerMode:storeKitVersion:)")
+    @available(macOS, obsoleted: 1,
+               message: "Explicitly setting the StoreKit version is now required.",
+               renamed: "configure(withAPIKey:appUserID:observerMode:storeKitVersion:)")
```

Comment:
may be we could have the message explicitly say "for observer mode", i.e.: `Explicitly setting the StoreKit version is now required when using observer mode.`